run-tests.sh

```diff
@@ -20,4 +20,7 @@ pip freeze | grep -qi /brainiak || {
     exit 1
 }
 
-py.test --cov=brainiak
+sleep 10     # allow the run-time build of cython to complete
```

Comment:
This probably belongs to another file. Nothing is happening in `run-tests` at this time.